api/config/routes.rb

```diff
@@ -40,6 +40,8 @@
           put :cancel
         end
       end
+
+      resources :customer_returns
```

Comment:
Can you exclude the `:destroy` action from here? Otherwise it will also generate a `DELETE` route that raises an error. In fact, we have a ton of such "ghost" routes in the API unfortunately...

Reply:
Yep, good point - thanks!